src/context/consultation-context.tsx

```diff
@@ -45,11 +44,17 @@ export function useSavedConsultationNotes() {
   }
 }
 
+async function getProviderUuid() {
+  const response = await getApiCall(sessionUrl)
+  return response?.currentProvider?.uuid
+}
+
 function ConsultationContextProvider({children}) {
   const [patientDetails, setPatientDetails] = useState<PatientDetails>()
   const [patientUuid, setPatientUuid] = useState('')
   const [locationUuid, setLocationUuid] = useState('')
   const [savedConsultationNotes, setSavedConsultationNotes] = useState('')
+  const providerUuidRef = useRef('')
```

Comment:
Can we just create a variable for providerUuid not just a ref

Reply:
We need useRef so that with each render we can maintain previous value. But with local variable without hook value will be re-assigned on each render